app/main.py

```diff
@@ -1,3 +1,67 @@
+from __future__ import annotations
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: int | float) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: Distance | float | int) -> Distance:
+        if isinstance(other, Distance):
+            return Distance(self.km + other.km)
+        elif isinstance(other, (int, float)):
+            return Distance(self.km + other)
+        else:
+            raise ValueError("Unsupported operand type")
+
+    def __iadd__(self, other: Distance) -> Distance:
+        if isinstance(other, Distance):
+            self.km += other.km
+        elif isinstance(other, (int, float)):
+            self.km += other
+        return self
+
+    def __mul__(self, other: Distance) -> Distance:
+        return Distance(self.km * other)
+
+    def __truediv__(self, other: Distance) -> Distance:
+        if isinstance(other, Distance):
+            return NotImplemented
+        elif isinstance(other, (int, float)):
+            distance = self.km / other
+        return Distance(round(distance, 2))
+
+    def __lt__(self, other: Distance) -> bool:
+        if isinstance(other, Distance):
+            return self.km < other.km
+        elif isinstance(other, (int, float)):
+            return self.km < other
```

Comment:
YOu always must return bool
You can raise exception or just do if-else
```
        if isinstance(other, Distance):
            return self.km < other.km
        else:
            return self.km < other
```


Fix in all places